gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 # Minecraft expectations
-paper = "1.21-R0.1-SNAPSHOT"
+paper = "1.21.5-R0.1-SNAPSHOT"
```

Comment:
can we do that without breaking binary compatibility with older versions? (due to enum -> interface changes)

Reply:
Mm I guess probably not

Reply:
```
warning: using incubating module(s): jdk.incubator.vector
/home/runner/work/FastAsyncWorldEdit/FastAsyncWorldEdit/worldedit-bukkit/src/main/java/com/sk89q/worldedit/util/formatting/text/adapter/bukkit/SpigotAdapter.java:41: error: cannot find symbol
import net.md_5.bungee.chat.ChatVersion;
                           ^
  symbol:   class ChatVersion
  location: package net.md_5.bungee.chat
/home/runner/work/FastAsyncWorldEdit/FastAsyncWorldEdit/worldedit-bukkit/src/main/java/com/sk89q/worldedit/util/formatting/text/adapter/bukkit/SpigotAdapter.java:43: error: cannot find symbol
import net.md_5.bungee.chat.VersionedComponentSerializer;
                           ^
```

Reply:
Fixed it by using some more reflection